org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -1,16 +1,17 @@
 /* jshint esversion: 6 */
 /* jshint node: true */
 'use strict';
+var OpenT2TErrorClass = require('opent2t').OpenT2TError;
```

Comment:
>var [](start = 0, length = 3)

SHould probably be 'let' -- wondering if there is a linter setup that i can run through the code and catch these errors -- have asked Chuck about it